src/components/ResourceList/ResourceList.tsx

```diff
@@ -474,9 +480,22 @@ class ResourceList extends React.Component<CombinedProps, State> {
       <div className={styles['HeaderWrapper-overlay']} />
     ) : null;
 
-    const showEmptyState = filterControl && !this.itemsExist() && !loading;
+    const showEmptyStateMarkup =
+      !loading && showEmptyState && emptyState && !this.itemsExist();
```

Comment:
I feel like it could lead to confusion to allow the consumer to determine whether they should show the empty state and then internal logic could override that. What do you think about `showEmptyState` taking precedence over everything else? Or even if an `emptyState` is defined show the empty state?

Reply:
> if an emptyState is defined show the empty state?

Ahhh that's what you meant in the other comment! Consumers can use the same conditional they'd use to set the value of `showEmptyState` as a ternary condition for setting the value of `emptyState`. I'll refactor and update the example and prop description 👍